autobuild/build.sh

```diff
@@ -107,12 +103,21 @@ fi
 if [[ `uname` == "Darwin" ]]; then
 	# autopoint is not in PATH after install via brew (conflicting with system gettext https://github.com/Homebrew/legacy-homebrew/issues/24070)
 	# so we can do `brew link --force gettext` or just add it to PATH before configuring which is preferable because we need it only for compiling
-	export PATH="/usr/local/opt/ccache/libexec:/usr/local/opt/gettext/bin:${PATH}"
-	export LDFLAGS="-L/usr/local/opt/gettext/lib ${LDFLAGS}"
-	export LDFLAGS="-L$(brew --prefix libomp)/lib ${LDFLAGS}"
-	export LDFLAGS="-L$(brew --prefix libtool)/lib ${LDFLAGS}"
-	export CPPFLAGS="-I/usr/local/opt/gettext/include ${CPPFLAGS}"
-	export PKG_CONFIG_PATH="/usr/local/opt/libffi/lib/pkgconfig:${PKG_CONFIG_PATH}"
+	export PATH="$(brew --prefix ccache)/libexec:$(brew --prefix gettext)/bin:${PATH}"
+	# `include` is required to find library headers. For example, currently we are using #include <harfbuzz/hb.h> instead of <hb.h>
+	# but pkg-config returns `harfbuzz/include/harfbuzz`, not just `harfbuzz/include`. This works fine on Linux because
+	# all system libraries is already included by compiler. This need to be fixed. (TODO)
+	export CPPFLAGS="-I$(brew --prefix)/include ${CPPFLAGS}"
+	# `lib` is required to correctly detect libpng/libjpeg in autotools. Current `configure.ac` script doesn't use
+	# `pkg-config` to get library flags.
```

Comment:
Want me to fix it first or we should leave as it is?

Reply:
I think better to fix this.

Reply:
I discovered `libjpeg` only provides a `.pc` file since version 1.5.0, that was released in 2016.
Is there any problem or should I mix with AC_CHECK_LIB?

PNG has `.pc` since April 2008 (at least)

MNG still doesn't provide it :P

Reply:
PNG fixed : #2443 
JPEG not done yet (waiting for opinion here:

> I discovered `libjpeg` only provides a `.pc` file since version 1.5.0, that was released in 2016. Is there any problem or should I mix with AC_CHECK_LIB?

)

Reply:
> I discovered libjpeg only provides a .pc file since version 1.5.0, that was released in 2016.
Is there any problem or should I mix with AC_CHECK_LIB?

I guess even in old-old debian releases it's already used. So I think we can only use pkg-config. 

> MNG still doesn't provide it :P

That's strange. On Mint 19 (Ubuntu 18.04) it's already here.

Reply:
> That's strange. On Mint 19 (Ubuntu 18.04) it's already here.

Official releases don't :( http://libmng.com/download.php?cat=25

Reply:
https://sourceforge.net/projects/libmng/files/libmng-devel/  OK. Version 2.0.2 (from August 2013) provides it, but it is not listed on their website :P

Reply:
But Debian still doesn't provide it o.O https://packages.debian.org/search?keywords=libmng
Libmng 2 is still on 1.10 :P

Reply:
Ok, got it. Ubuntu provides 2.0.2 since 18.04, Debian is still on 1.0.10.
https://pkgs.org/search/?q=libmng